gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -336,6 +335,16 @@ gst_tensor_src_iio_class_init (GstTensorSrcIIOClass * klass)
           "Mode for the device to run in - one-shot or continuous",
           DEFAULT_OPERATING_MODE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_BASE_DIRECTORY,
+      g_param_spec_string ("base-dir", "Base Dir",
+          "Base directory for IIO devices", DEFAULT_PROP_BASE_DIRECTORY,
+          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+
+  g_object_class_install_property (gobject_class, PROP_DEV_DIRECTORY,
```

Comment:
I am not sure that a property to indicate "/dev/".

Reply:
As long as we have a sound default value, I guess it's ok. We may have a pervert who uses /ved/ for device files :)

Reply:
@myungjoo Understood. Thanks for the explanation:)